apps/docs/src/app/core/component-docs/combobox/combobox-docs.component.html

```diff
@@ -41,6 +41,26 @@
 
 <separator></separator>
 
+<fd-docs-section-title [id]="'combobox-mobile'" [componentName]="'combobox'">
+    Combobox Mobile Mode
+</fd-docs-section-title>
+<description>
+    There is way to use mobile mode on combobox component. To achieve it user needs to:
+    <ul>
+        <li>Set <code>[mobile]</code> property to true</li>
+        <li>Provide <code>DIALOG_CONFIG</code></li>
```

Comment:
what if when user provides dialog config it automatically sets mobile to true and mobile is a hidden property. Since he is forced to provide both might as well only force user to provide one

Reply:
Dialog config doesn't have impact on fact if mobile is set on combobox.